rpc/query_server.go

```diff
@@ -1167,21 +1167,25 @@ func getReceiptByTendermintHash(
 }
 
 func completeReceipt(txResults *ctypes.ResultTx, blockResult *ctypes.ResultBlock, txReceipt *types.EvmTxReceipt) *eth.JsonTxReceipt {
+	jsonReceipt := eth.EncTxReceipt(*txReceipt)
```

Comment:
Moving it up here means the block time won't be set in the returned receipt, doesn't it?

Reply:
Move it back